components/tools/OmeroWeb/omeroweb/webclient/controller/container.py

```diff
@@ -284,7 +305,7 @@ def listFigureScripts(self, objDict=None):
             'tooltip': "Create a movie of the image"}
         if (self.image and (self.image.getSizeT() > 0 or
                             self.image.getSizeZ() > 0)):
```

Comment:
@dominikl Not related to this PR, but I think we should only enable Movie if it's a multiple image: The ```sizeT > 1 or sizeZ > 1```.